### PR TITLE
Having >300% blood will gib you.

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Having >300% blood will gib you.
Realistically this will only occur if you get yourself over 1680u of blood using IVs pre-drop and quickclot.
And it will trigger if your QC gets purged.
## Why It's Good For The Game
Personally I think gibbing humans is quite funny.
## Changelog
:cl:
balance: Having >300% Blood and no QC will gib you
/:cl:

### CODE REVIEW

File: code/modules/mob/living/blood.dm [hunk 1]
```diff
@@ -16,22 +16,30 @@
...
 						to_chat(src, span_notice("Some spare blood leaks out of your nose."))
 					if(30 to 100)
 						to_chat(src, span_notice("Spare blood gushes out of your ears and mouth. Must've had too much."))
 						bleed_range = 1
-					if(100 to INFINITY)
+					if(100 to 300)
 						visible_message(span_notice("Several jets of blood open up across [src]'s body and paint the surroundings red. How'd [p_they()] do that?"), \
 							span_notice("Several jets of blood open up across your body and paint your surroundings red. You feel like you aren't under as much pressure any more."))
 						bleed_range = 3
+					if(300 to INFINITY)
```
Comment (QualityVan) [inline code/modules/mob/living/blood.dm:30]: spare_blood isn't the percent blood you have, it's the total units you have over 580.

**Comment by Lumipharon** [comment]
Being able to get to ridiculously high blood % is a meme to begin with tbh

**Comment by Arctrooper8** [comment]
Why not just make it so quick clot doesn't prevent the blood from spurting out of you.

**Comment by 00ze-cyclone** [comment]
This is another wrench thrown in the setup of nanite users but at the same time
Funni gib

**Comment by FosterGrime-Heymynameslouie** [comment]
how do you stuff yourself with 3x the normal amount of blood a person should have?

**Comment by TiviPlus** [comment]
> I mean you could just kill QC giving you no blood limit

Yea but its funny

**Comment by TiviPlus** [comment]
https://tenor.com/view/dr-manhattan-gif-18899941

**Comment by lbnesquik** [comment]
Is there any sort of warning that you are going to straight up get gibbed?

**Comment by Xander3359** [comment]
> Is there any sort of warning that you are going to straight up get gibbed?

No. If your quick clot runs out (or you get purged), you will just gib suddenly :)

File: code/modules/mob/living/blood.dm (hunk 1 above)
Comment (Xander3359) [reply code/modules/mob/living/blood.dm:30]: Oh my mistake, I thought it was in %, I'll change it to the appropriate number

**Comment by Xander3359** [comment]
> Why not just make it so quick clot doesn't prevent the blood from spurting out of you.

Quick clot does prevent blood from spurting out. That is how you reach insane amounts of blood% in the first place.
The only way to trigger this is to have quick clot and get this much blood in you, and then let the QC expire.

**Comment by novaepee** [comment]
> https://tenor.com/view/dr-manhattan-gif-18899941

dr-manhattan-143519013

**Comment by Arctrooper8** [comment]
> > Why not just make it so quick clot doesn't prevent the blood from spurting out of you.
> 
> Quick clot does prevent blood from spurting out. That is how you reach insane amounts of blood% in the first place. The only way to trigger this is to have quick clot and get this much blood in you, and then let the QC expire.

Yes so my point is: Why not just remove that feature. Quick Clot keeping blood in you is only used by nanite users to stack blood to absurd levels.

**Comment by Xander3359** [comment]
> > > Why not just make it so quick clot doesn't prevent the blood from spurting out of you.
> > 
> > 
> > Quick clot does prevent blood from spurting out. That is how you reach insane amounts of blood% in the first place. The only way to trigger this is to have quick clot and get this much blood in you, and then let the QC expire.
> 
> Yes so my point is: Why not just remove that feature. Quick Clot keeping blood in you is only used by nanite users to stack blood to absurd levels.

I think Quick-Clot allowing you to go over 100% Blood level is very cool.
I also think nanite users who fly too close to the sun around defilers getting gibbed is also very cool.
I say this as someone who uses nanites and has done the QC strat a few times.
If you think the punishment is too much, ask your local doctor to tell you what your blood is, or just don't stay hooked up to 5 IVs for 5 minutes, 300% blood level is more than enough to last you an entire round.

**Comment by Mrrpip** [comment]
Time to troll with hypervene. :D

**Comment by Naaanii** [comment]
Why is my CMO gaming specifically being nerfed? Smh

**Comment by srxtreme** [comment]
Based

**Comment by Tommy-211** [comment]
just know this wont stop shit lmao

**Comment by Tommy-211** [comment]
nothing screams fun and riveting gameplay like randomly exploding.

**Comment by chizzy376** [comment]
Yes let's give bald corpsmen another reason to be pb'd for using hypervene

**Comment by Xander3359** [comment]
> Yes let's give bald corpsmen another reason to be pb'd for using hypervene

You can't PB them if you're gibbed :)

**Comment by Tommy-211** [comment]
> > Yes let's give bald corpsmen another reason to be pb'd for using hypervene
> 
> You can't PB them if you're gibbed :)

https://www.youtube.com/watch?v=xBWmkwaTQ0k

**Comment by Xander3359** [comment]
Maybe the real blood code was the friends we made along the way.

**Pull request closed without merging**